matchms/similarity/PrecursormzMatch.py

```diff
@@ -0,0 +1,127 @@
+from typing import List
+import numba
+import numpy
+from matchms.typing import SpectrumType
+from .BaseSimilarity import BaseSimilarity
+
+
+class PrecursormzMatch(BaseSimilarity):
+    """Return True if spectrums match in precursor m/z (within tolerance), and False otherwise.
+
+    Example to calculate scores between 2 spectrums and iterate over the scores
```

Comment:
I see 4 spectrums or 2 pairs of spectrums 
```suggestion
    Example to calculate scores between 2 pairs of spectrums and iterate over the scores
```

Reply:
True, that's better.